influxql/engine.go

```diff
@@ -569,7 +577,7 @@ func NewPlanner(db DB) *Planner {
 
 // Plan creates an execution plan for the given SelectStatement and returns an Executor.
 func (p *Planner) Plan(stmt *SelectStatement) (*Executor, error) {
-	now := p.Now()
+	now := p.Now().UTC()
```

Comment:
As far as I understand time, this isn't an error, but is useful for when doing debugging that you always see everything in UTC.  Internally, the epoch time (int64) is the same if you ask for UTC or not.